.github/workflows/continuous_deployment.yml

```diff
@@ -146,6 +146,7 @@ jobs:
 
     - name: Prepare namespace
       run: |
+        kubectl label ns "${NAMESPACE}" deployment=sara || true
```

Comment:
What is this for?

Reply:
This label is added to ensure that the namespace has at least one label, which helps `rasactl` to request namespace information properly. 

`kubectl` won't give you the requested json data if the namespace does not have any label. 🤷🏻‍♀️

Reply:
It was a workaround, a bug was fixed in the latest version of rasactl. @HotThoughts we can remove this line, it's not needed anymore.

Reply:
```suggestion
```

Reply:
Wow. That was fast. Thank you @tczekajlo for fixing it. I will remote this line.